src/test/java/seedu/address/logic/parser/AddClientCommandParserTest.java

```diff
@@ -9,21 +9,18 @@
 import static seedu.address.logic.commands.CommandTestUtil.INVALID_EMAIL_DESC;
 import static seedu.address.logic.commands.CommandTestUtil.INVALID_NAME_DESC;
 import static seedu.address.logic.commands.CommandTestUtil.INVALID_PHONE_DESC;
-import static seedu.address.logic.commands.CommandTestUtil.INVALID_TAG_DESC;
```

Comment:
is there supposed to be invalid_product_desc also? seems it should be a refactor of `INVALID_TAG_DESC`.